lib/api/client-commands/window/setPosition.js

```diff
@@ -1,4 +1,5 @@
-const ProtocolAction = require('../_base-action.js');
+const ClientCommand = require('../_base-command.js');
+const {Logger} = require('../../../utils');
```

Comment:
There are mutliple references of `Logger` in files which are not used. Should we remove those?